util/libcrypto.num

```diff
@@ -5513,3 +5513,4 @@ ASN1_item_unpack_ex                     ?	3_2_0	EXIST::FUNCTION:
 PKCS12_SAFEBAG_get1_cert_ex             ?	3_2_0	EXIST::FUNCTION:
 PKCS12_SAFEBAG_get1_crl_ex              ?	3_2_0	EXIST::FUNCTION:
 EC_GROUP_to_params                      ?	3_2_0	EXIST::FUNCTION:EC
+OSSL_CMP_SRV_CTX_setup_polling          ?	3_2_0	EXIST::FUNCTION:CMP
```

Comment:
Correct. 
Yet please move just after the last `OSSL_CMP_` function above, such that all new CMP functions are at one place.
This may also help avoiding merge conflicts when new functions for other parts of the lib are added.